homeassistant/components/tuya/climate.py

```diff
@@ -131,6 +133,7 @@ def __init__(
         """Determine which values to use."""
         self._attr_target_temperature_step = 1.0
         self.entity_description = description
+        self._is_thermostat = device.category == "wk"
```

Comment:
We should not hardcode this. Instead, handle this in the entity descriptions.

Reply:
Check my below comment. This flag was introduced in order to lower effect of my changes to other device that may have better support for modes

Reply:
Removed